pages/doc/integrations_tas_custom_app_metrics_howto.md

```diff
@@ -0,0 +1,75 @@
+
+**We recommend the Metrics Registrar approach** as a simpler and more predictable pattern, but teams may choose either option. Both have drawbacks, and for some workloads only the push method can be used in practice.
+
+## Push vs Pull Models Generally
+
+The “pull” or “scrape” model has gained popularity in recent years. The typical pull-based design involves hosting an HTTP route on your server, typically at `/metrics`, which exposes custom metrics. Another process periodically fetches the response from that route and transforms it into metrics within a metric store.
+
+In the push model, the application instead periodically sends metrics directly to a server using authentication credentials provided to the application.
+
+Scraping is a popular approach, because the application being monitored does not need to know anything about the metrics store, such as an endpoint or a credential.
+
+In the scraping model, the rate of metrics collection can be centrally controlled. In a push model, applications may or may not be able to throttle their metric volume if they are overwhelming the metrics service.
+
+The output of a `/metrics` endpoint in a scraping model is normally human readable, which can help with debugging the metrics coming from a running application.
+
+Pushing metrics can produce more detail; in the push model, a metric can be submitted as often as practically limits allow - possibly multiple times per second. In a scrape model, only changes since the last scrape are seen. Typically metrics data does not need to be extremely high-fidelity in order to be useful, and this periodic sampling is acceptable to most users.
+
+Very short-lived processes **must** use a push model, because they do not run long enough to be discovered and scraped by a metrics collector.
+
+## Metrics Registrar
+
+TAS supports a pull model for custom app metrics using the “Metrics Registrar” component.
+
+The metrics registrar is a TAS component that periodically scrapes registered endpoints and forwards scraped metrics to the Loggregator Firehose. In the Wavefront Nozzle, these metrics are then ingested in the same way as platform or service tile metrics, and become visible in Wavefront automatically.
+
+To instrument an application for metrics registrar scraping, you will typically use either a prometheus SDK or an metrics framework built into your application stack, such as Spring Boot Actuator.
+
+After deploying your application, you register it using the `cf` CLI, as described in the documentation: [https://docs.pivotal.io/application-service/metric-registrar/using.html](https://docs.pivotal.io/application-service/metric-registrar/using.html).
+
+## Wavefront Service Broker
+
+To implement a push-based metrics pattern using Wavefront, you can use the Wavefront SDK for your language stack, or you can use a pluggable abstraction like Spring Boot Actuator.
```

Comment:
To implement a push-based metrics method by using Tanzu Observability, you can use the Wavefront SDK for your language stack, or you can use a pluggable abstraction such as Spring Boot Actuator.

In either case, you must provide your application with a URL for a Wavefront proxy instance.

The supported way to do this using the Tanzu Observability by Wavefront Nozzle is to use the Wavefront Service Broker, create a service instance, and bind it to your application. You can then push custom metrics to the proxy from your own application code.

Reply:
I made all of the changes here except for "pattern" => "method". 

I see how "method" signifies "model" or "approach here. However I felt hesitant because the word method also reminds me of a method in a program.

What do you think about going back to the word "model" from earlier in the text, instead of "pattern" or "method"? 

However happy to defer to for the final call, @Margarita-Staneva.

Reply:
I'd use model. :-)